pkg/dns/aws/dns.go

```diff
@@ -78,6 +80,10 @@ func NewManager(config Config) (*Manager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't create AWS client session: %v", err)
 	}
+	sess.Handlers.Build.PushBackNamed(request.NamedHandler{
+		Name: "openshift.io/cluster-ingress-operator",
```

Comment:
can you make this `openshift.io/ingress-operator`

Reply:
done with https://github.com/openshift/cluster-ingress-operator/compare/a60a53d976a6f8b9312678fdd5582b4716373a5c..b8dfa91aa6707cd3a3af790e4e6524821973034d